Tests/Resources/TemplateRenderingTest.php

```diff
@@ -239,6 +239,57 @@ public function testStepList_stepSkipped() {
 		$this->assertContains('<li class="craue_formflow_current_step">step2</li>', $renderedTemplate);
 	}
 
+	public function testRenderResetButton() {
```

Comment:
Please rename this function to `testButtons_noResetButton`.
And would you move it right below `testButtons`? It fits better there.